src/OrchardCore.Modules/OrchardCore.Resources/wwwroot/Styles/jquery-ui.css

```diff
@@ -1310,3 +1310,8 @@ a.ui-button:active,
 	-webkit-box-shadow: 0px 0px 5px #666666;
 	box-shadow: 0px 0px 5px #666666;
 }
+
+/* Overrides datepicker */
```

Comment:
You can't change this file, it's supposed to be the original jquery source code, it will get erased eventually by a new version.

Reply:
That's why I am asked at the beginning there's no entry for it in `Assets.json`

Reply:
@hishamco You can add style overrides into the TheAdmin theme. This way we keep those libraries as they are distributed.